create-update.sh

```diff
@@ -102,6 +102,13 @@ if [  "$KEEP_CHROOTS" -eq 0 ]; then
 		done
 fi
 
+# step 1.5: sign the Manifest.MoM that was just created
+if [ -f "ClearLinuxRoot.pem" ]; then
+		sudo openssl smime -sign -binary -in update/www/$MIXVER/Manifest.MoM \
```

Comment:
Thanks for the comments, and yes the fields (date, subj, etc metadata) will be configurable soon, I just wanted this to be a first pass at enabling the feature so users who just want it to work - have it working.